lte/gateway/c/oai/oai_mme/oai_mme.c

```diff
@@ -139,6 +139,7 @@ int main(int argc, char* argv[]) {
   event_client_init();
 
   CHECK_INIT_RETURN(mme_app_init(&mme_config));
+  CHECK_INIT_RETURN(amf_app_init(&amf_config));
```

Comment:
AMF task should be started only if converged core is enabled (similar to NGAP task).